server/database/dataGeneration.js

```diff
@@ -0,0 +1,109 @@
 XOXO YOLO mlkshk 90\'s messenger bag.', 'Bushwick brunch craft beer man braid literally selvage XOXO.', 'Gentrify YOLO waistcoat lyft raw denim, selfies organic artisan.', 'Squid green juice poutine iceland shoreditch selfies.', 'Gentrify iceland subway tile godard adaptogen drinking vinegar chillwave franzen glossier viral edison bulb.', 'Lyft leggings lomo pitchfork pork belly tattooed.', 'Hexagon affogato seitan jean shorts live-edge schlitz.', 'Mumblecore kogi waistcoat, af edison bulb photo booth thundercats offal hammock cliche hashtag sriracha.', 'I\'m baby meditation synth tbh vinyl iceland gentrify art party forage. DIY chartreuse microdosing lumbersexual ethical keffiyeh.', 'Pour-over succulents polaroid, tacos fashion axe flexitarian vape synth affogato artisan meggings selfies unicorn gluten-free.', 'Plaid portland taxidermy shabby chic bicycle rights migas flannel narwhal shoreditch poke pickled selvage adaptogen everyday carry.', 'Photo booth intelligentsia enamel pin scenester, woke snackwave authentic man bun portland.', 'Master cleanse succulents irony pitchfork green juice organic.', 'Kinfolk vice meh PBR&B synth, tattooed air plant post-ironic shoreditch quinoa tofu hoodie bespoke fixie thundercats.', 'Paleo pinterest yuccie you probably haven\'t heard of them affogato tbh.', 'Skateboard heirloom hella, humblebrag hammock lyft pork belly kinfolk food truck locavore hexagon chambray kickstarter butcher.', 'Hell of godard banjo tousled sartorial, wayfarers freegan DIY.', 'Drinking vinegar kombucha post-ironic fashion axe single-origin coffee listicle glossier intelligentsia.', 'Humblebrag af occupy literally hexagon, sustainable poke fingerstache gluten-free.', 'Thundercats etsy cred gastropub tote bag succulents.', 'Glossier sustainable jianbing migas artisan lumbersexual kitsch humblebrag listicle mustache gluten-free cloud bread lyft tofu asymmetrical.', 'Kombucha PBR&B health goth, hot chicken cornhole keffiyeh next level raclette typewriter pork belly man braid.', 'Humblebrag lo-fi adaptogen affogato bespoke freegan gluten-free activated charcoal helvetica flexitarian.', 'Affogato next level seitan, meh pop-up cold-pressed vice pour-over wolf.', 'Truffaut art party iPhone vinyl everyday carry af mixtape cornhole williamsburg mustache salvia affogato.', 'Intelligentsia viral taiyaki brooklyn, la croix shabby chic plaid master cleanse try-hard VHS kombucha jean shorts fanny pack cliche.', 'Lomo blog microdosing celiac, kombucha cliche iPhone cloud bread meh succulents.', 'Meditation ramps stumptown mlkshk kitsch aesthetic slow-carb shoreditch everyday carry blue bottle.', 'Everyday carry hell of heirloom, four dollar toast pitchfork fingerstache celiac cornhole locavore squid crucifix kogi waistcoat direct trade.', 'Fixie direct trade semiotics kinfolk pug irony unicorn mustache, butcher stumptown sustainable DIY master cleanse gastropub blue bottle.', 'Celiac normcore health goth snackwave viral hoodie cloud bread tumblr.'];
```

Comment:
I think Faker can generate fake sentences for you rather than having to hard-code an array